IdleServer.js

```diff
@@ -40,21 +40,12 @@ const SERVER_CONFIG = {
 		"3D16B": "http://UltrabitWebServicesProd.ya4zvdebvk.us-west-2.elasticbeanstalk.com/", // kongregate/production
 		"19081": "http://uws-dev-env.ddhjvp3mrf.us-west-1.elasticbeanstalk.com/", // staging
 		"18A45": "http://uws-dev-env.ddhjvp3mrf.us-west-1.elasticbeanstalk.com/"  // auto
-	}
-};
-
-const PLAYFAB_ENVIRONMENT_CONFIG = {
-	configs: {
-		"18A45": {
-			apiKey: "ORX7Y8EKSK5DD1MOKUE86X5D64RQ774359ICEJJMN7U176GOBF", // playfab auto
-			logglyTag: 'idle-spongebob-auto-ops'
-		},
-		"3D16B": {
-			apiKey: "9QPQ1B5Q64G8Q3DAJE9WUFTC957UP3MZADYICBD97SWBXG853J",   // playfab kongregate
-			logglyTag: 'idle-spongebob-kongregate-ops'
-		}
 	},
-	playfabIds: ["3D16B", "18A45"]
+	logglyTag: {
+                "3D16B": "idle-spongebob-kongregate-ops", // kongregate/production
+                "19081": "idle-spongebob-auto-ops", // staging
+                "18A45": "idle-spongebob-auto-ops"  // auto
```

Comment:
I believe you will also need to create the respective tags in Loggly

Reply:
Already done!  :D